core/src/main/java/org/elasticsearch/action/support/IndicesOptions.java

```diff
@@ -120,12 +121,16 @@ public boolean ignoreAliases() {
     }
     
     public void writeIndicesOptions(StreamOutput out) throws IOException {
-        out.write(id);
+        if (out.getVersion().onOrAfter(Version.V_6_0_0_alpha1_UNRELEASED)) {
```

Comment:
not sure why this is still mark unreleased, we have already released alpha1, I think you should use alpha2.